src/withPayline.tsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react';
+import React, { PropsWithChildren } from 'react';
 import PaylineProvider from './PaylineProvider';
 
-const withPayline = (production: boolean) => (WrappedComponent: React.ComponentType) => (props: any) => (
+const withPayline = (production: boolean) => <TProps,>(WrappedComponent: React.ComponentType<TProps>) => (props: PropsWithChildren<TProps>) => (
```

Comment:
Why the trailing comma? Is it a typo?

Reply:
Hi, it's not a typo, it's quirk of Typescript with JSX

Reply:
I will split the PT asap